src/beta_inc.jl

```diff
@@ -9,8 +9,9 @@ const exparg_p =  log(prevfloat(floatmax(Float64)))
 
 Computes ``log(\\Gamma(b)/\\Gamma(a+b))`` when b >= 8
 """
-loggammadiv(a::Number, b::Number) = _loggammadiv(float(a), float(b))
+loggammadiv(a::Number, b::Number) = _loggammadiv(promote(float(a), float(b))...)
 
+_loggammadiv(a::Number, b::Number) = loggamma(b) - loggamma(a + b)
```

Comment:
I think it would be useful to add a comment here as a reminder that a proper implementation should be added at some point.

Reply:
I added a comment.